python/src/GeometricProfile_doc.i.in

```diff
@@ -36,22 +33,23 @@ Examples
 OT_GeometricProfile_doc
 // ---------------------------------------------------------------------
 %define OT_GeometricProfile_call_doc
-"Compute the temperature T(i).
+"Compute the temperature :math:`T(i)`.
 
 Parameters
 ----------
-i : int
-    The index for whome we seek temperature
+i : int,
+    The index at which the temperature is computed.
 
 Returns
 -------
-temperature:  float
-    The temperature
+temperature:  float,
```

Comment:
no comma after, and a whitespace before and after :

Reply:
ok